components/style/values/specified/basic_shape.rs

```diff
@@ -768,23 +768,22 @@ impl Parse for BorderRadius {
     }
 }
 
-fn parse_one_set_of_border_values(context: &ParserContext, mut input: &mut Parser)
+fn parse_one_set_of_border_values(_context: &ParserContext, mut input: &mut Parser)
```

Comment:
One final change. I didn't notice this earlier. This function is private, and it's used only by `BorderRadius`. So, let's pass only the `Parser` to this function (which will make `BorderRadius::parse` ignore the `ParserContext`.